apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -46,6 +46,7 @@ defmodule OMG.Watcher.ExitProcessor do
   Returns `db_updates`
   """
   def new_exits(exits) do
+    Appsignal.increment_counter("exit.started", length(exits))
```

Comment:
I think all of these (and more) can be handled by a single line in `EthereumEventListener` - you count by the listener name (`service name`) and increment by `length(events)`, when they are consumed. Seems like the effect would be even richer then (you're missing out deposits)